docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,37 @@
-### What was the problem?
+<!--- Provide a general summary of your changes in the Title above -->
 
-### How did I solve it?
+### What issue have I solved?
+<!--- Complementary description if needed -->
+Resolves #INSERT_ISSUE_NUMBER
 
-### How to manually test it?
+### How have I implemented/fixed it?
+<!--- Describe your technical implementation -->
 
-### Review checklist
 
-- [ ] The PR resolves #INSERT_ISSUE_NUMBER
-- [ ] All new code is covered with unit tests
-- [ ] Relevant integration / functional tests are added
+### How has this been tested?
+<!--- Please describe how you tested your changes -->
+
+
+### Definition of Done
+- [ ] Coding done as per the guidelines
+- [ ] Complete unit testing
+- [ ] Complete functional testing
+- [ ] Complete integration testing
+- [ ] Complete code review
+- [ ] No major bugs pending
+- [ ] Documents updated (if any)
+- [ ] Acceptance test by lead and approved
```

Comment:
is the lead necessary? I think we can remove that and just keep `Acceptance test covered`?